elevate-cpanel

```diff
@@ -2020,6 +2090,32 @@ sub kernel_check ($self) {
     return;
 }
 
+sub backup_eaphp_configs {
```

Comment:
How long will it take you to merge the changes, approximately? Just to ascertain if I can throw out this modification immediately.

Reply:
We'll probably be working on a version of this commit in the next couple of weeks. This PR is ineligible at this point because elevate-cpanel is now generated. We'll take care of the re-factor once we understand the code.